pkg/util/config.go

```diff
@@ -17,15 +17,18 @@ limitations under the License.
 package util
 
 import (
+	"k8s.io/klog/v2"
 	"k8s.io/utils/pointer"
 
+	"github.com/koordinator-sh/koordinator/apis/extension"
```

Comment:
we prefer renaming the package into `apiext`

Reply:
Let's open another PR to rename this package.

Reply:
Let's rename this package in all files with another PR.

Reply:
ok